src/js/common/components/Challenge/ChallengeAbout.jsx

```diff
@@ -138,30 +148,7 @@ function ChallengeAbout ({ challengeWeVoteId, showDaysLeft }) {
             </FlexDivLeft>
           </Suspense>
         </CardForListRow>
-        {showStartedBy && (
```

Comment:
Please maintain the same place in the code for your new challengeOwners changes, as it was prior to your changes (i.e, the top-to-bottom order). Please support the "showStartedBy" variable -- there are some modes where we won't show this started by section.